config/locales/ptbr.yml

```diff
@@ -475,6 +475,7 @@ ptbr:
         mindfulness: paz de espírito
         journaling: Diário
         mood_tracking: Registro de alterações de humor
+	veterans: veteranos
```

Comment:
Looks like the CI build is failing because this line is indented by tabs instead of spaces?

Reply:
Thank you. Fixed.